ads/mywidget.md

```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# myWidget
+
+myWidget is a service for personalizing content and increasing user metrics.
+
+Our NeuroClick technology uses behavioral data of eighty million users, collaborative filtering algorithms and neural networks. NeuroClick delivers the most relevant content to every user of the site.
+
+myWidget is a free platform used by web publishers to improve metrics and audience loyalty.
+
+Please visit our [website](https://widget.my.com) for more information about us.
+
+## Example
+
+```html
+<amp-embed height="250"
+      type="mywidget"
+      data-cid="your_campaign_id">
+  <div placeholder>Loading myWidget recommendations.</div>
+  <div fallback>No recommendations for you.</div>
+</amp-embed>
+```
+
+Supported parameters:
+
+- `data-cid`
```

Comment:
I see that `data-cid` is a required parameter,  I would suggest indicate that it is required here.